{"uuid": "b25eafef-0d5c-458e-9634-e79351e5cff8", "children": ["6831ad3c-50a9-44e4-990d-0d4c440ac957"], "befores": [{"name": "Valid_Login_With_login_Code", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nBacktrace:\n\t(No symbol) [0x008937D3]\n\t(No symbol) [0x00828B81]\n\t(No symbol) [0x0072B36D]\n\t(No symbol) [0x0075D382]\n\t(No symbol) [0x0075D4BB]\n\t(No symbol) [0x00793302]\n\t(No symbol) [0x0077B464]\n\t(No symbol) [0x00791215]\n\t(No symbol) [0x0077B216]\n\t(No symbol) [0x00750D97]\n\t(No symbol) [0x0075253D]\n\tGetHandleVerifier [0x00B0ABF2+2510930]\n\tGetHandleVerifier [0x00B38EC1+2700065]\n\tGetHandleVerifier [0x00B3C86C+2714828]\n\tGetHandleVerifier [0x00943480+645344]\n\t(No symbol) [0x00830FD2]\n\t(No symbol) [0x00836C68]\n\t(No symbol) [0x00836D4B]\n\t(No symbol) [0x00840D6B]\n\tBaseThreadInitThunk [0x75C700F9+25]\n\tRtlGetAppContainerNamedObjectPath [0x77C77BBE+286]\n\tRtlGetAppContainerNamedObjectPath [0x77C77B8E+238]\n\n", "trace": "  File \"C:\\Users\\user\\QA_Automation\\2023_Final_Project\\venv\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\user\\QA_Automation\\2023_Final_Project\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 1129, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\user\\QA_Automation\\2023_Final_Project\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 908, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\user\\QA_Automation\\2023_Final_Project\\TRADO\\QA_WEB\\Pages\\Pre_Requirment_page\\Pre_Requirement_login.py\", line 80, in Valid_Login_With_login_Code\n    self.enter_phone(Phone)\n  File \"C:\\Users\\user\\QA_Automation\\2023_Final_Project\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\user\\QA_Automation\\2023_Final_Project\\TRADO\\QA_WEB\\Pages\\Pre_Requirment_page\\Pre_Requirement_login.py\", line 50, in enter_phone\n    wait.until(EC.presence_of_element_located((By.XPATH, Pre_Requirement_Locators.PHONE_FIELD)))\n  File \"C:\\Users\\user\\QA_Automation\\2023_Final_Project\\venv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "click_restaurants", "status": "passed", "start": 1677103280756, "stop": 1677103280838}, {"name": "click_cocktails", "status": "passed", "start": 1677103280838, "stop": 1677103280900}, {"name": "click_save_button", "status": "passed", "start": 1677103280900, "stop": 1677103281302}, {"name": "click_login_section", "status": "passed", "start": 1677103281303, "stop": 1677103281476}, {"name": "enter_phone", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nBacktrace:\n\t(No symbol) [0x008937D3]\n\t(No symbol) [0x00828B81]\n\t(No symbol) [0x0072B36D]\n\t(No symbol) [0x0075D382]\n\t(No symbol) [0x0075D4BB]\n\t(No symbol) [0x00793302]\n\t(No symbol) [0x0077B464]\n\t(No symbol) [0x00791215]\n\t(No symbol) [0x0077B216]\n\t(No symbol) [0x00750D97]\n\t(No symbol) [0x0075253D]\n\tGetHandleVerifier [0x00B0ABF2+2510930]\n\tGetHandleVerifier [0x00B38EC1+2700065]\n\tGetHandleVerifier [0x00B3C86C+2714828]\n\tGetHandleVerifier [0x00943480+645344]\n\t(No symbol) [0x00830FD2]\n\t(No symbol) [0x00836C68]\n\t(No symbol) [0x00836D4B]\n\t(No symbol) [0x00840D6B]\n\tBaseThreadInitThunk [0x75C700F9+25]\n\tRtlGetAppContainerNamedObjectPath [0x77C77BBE+286]\n\tRtlGetAppContainerNamedObjectPath [0x77C77B8E+238]\n\n", "trace": "  File \"C:\\Users\\user\\QA_Automation\\2023_Final_Project\\venv\\lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n  File \"C:\\Users\\user\\QA_Automation\\2023_Final_Project\\TRADO\\QA_WEB\\Pages\\Pre_Requirment_page\\Pre_Requirement_login.py\", line 50, in enter_phone\n    wait.until(EC.presence_of_element_located((By.XPATH, Pre_Requirement_Locators.PHONE_FIELD)))\n  File \"C:\\Users\\user\\QA_Automation\\2023_Final_Project\\venv\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "parameters": [{"name": "phone_number", "value": "'3322114455'"}], "start": 1677103281476, "stop": 1677103303738}], "start": 1677103280755, "stop": 1677103303739}], "start": 1677103280755, "stop": 1677103304005}